src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/VsTestConsoleWrapper.cs

```diff
@@ -142,17 +142,12 @@ public void CancelDiscovery()
         /// <inheritdoc/>
         public void RunTests(IEnumerable<string> sources, string runSettings, ITestRunEventsHandler testRunEventsHandler)
         {
-            this.RunTests(sources, runSettings, new TestPlatformOptions(), testRunEventsHandler);
+            this.RunTests(sources, runSettings, null, testRunEventsHandler);
```

Comment:
Do we want users to use the below API when `TestPlatformOptions` is null? Would expect users to use this one in that case.

Reply:
We check for null down stream, so I figured there's no harm.

Reply:
Consider: Use named parameters for all these method call with null option.